### DISCUSSION TITLE
Timeout while connecting to copilot from IntelliJ IDE

### DISCUSSION THREAD

**Discussion by vipinchandranp** [community/community#121320] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

While logging into Github through copilot I am getting below error.

Sign in failed. Reason: Request signInInitiate failed with message: connect ETIMEDOUT XXXXX , request id: 2, error code: -32603

**Comment by vipinchandranp** [comment, marked as answer]
I think I got the fix. I had to check the box called "Show IDE completions side-by-side" in the copilot setting. I am not sure how it's related to the error which I saw (_Sign in failed. Reason: Request signInInitiate failed with message: connect ETIMEDOUT XXXXX , request id: 2, error code: -32603_). The error message is a bit misleading.

**Reply by boweipacer** [reply]
great, thanks for your tip. That's true.

**Comment by dh64867** [comment]
It helped me but when I restart IDE I have to uncheck box. I have to check box or uncheck box every time I launch IDE. 

That looks strange. Any suggestions?
